web/src/analytics.ts

```diff
@@ -0,0 +1,41 @@
+import { useConnectionState } from "@/data/hooks";
+import { setRtdmLocalStorage, getRtdmLocalStorage } from "@/local-storage";
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+import { v4 as uuidv4 } from "uuid";
+
+export function trackAnalyticsEvent(
+  event: string,
+  params?: Record<string, string | number>
+) {
+  window.analytics.track(`rtdm-${event}`, params);
+}
+
+export function identifyUser() {
+  let userID = getRtdmLocalStorage().userID;
```

Comment:
since you are just using this here, and thus just using it in useAnalytics, you can easily use a recoil atom to manage the user id and remove the local-storage module entirely